src/loader/shape_.mli

```diff
@@ -0,0 +1,16 @@
+open Odoc_model
```

Comment:
It's unclear that the new name is better. It's purpose did not change and the new alias is conflicting.

Reply:
You are rigth! Reverted.